### DISCUSSION TITLE
Sessions Management using Credentials Provider and Prisma ORM schema.

### DISCUSSION THREAD

**Discussion by paperpluto** [nextauthjs/next-auth#4334] [Help]
### Question 💬

How can I let user manage sessions(login activity / Browser sessions) by using Credentials provider with email verify link and showing device information time and session active status when user creates account and is logged in?

I am getting a little information from the documentation as it is limited on my issue I think.
I am new to this library so any extra information would help a lot. Thanks.

![image](https://user-images.githubusercontent.com/79751398/161449544-e41bd23d-5d7c-473a-a318-a66939ff8cb3.png)

![image](https://user-images.githubusercontent.com/79751398/161449205-3b7d826c-e6a8-4121-a6b7-3b080e1d0cf0.png)

### How to reproduce ☕️

![image](https://user-images.githubusercontent.com/79751398/161449252-f6c47b94-606e-4db6-9b0c-3dea2168c34e.png)
I want to make something like this.

### Contributing 🙌🏽

No, I am afraid I cannot help regarding this

**Comment by ThangHuuVu** [comment]
I'm not sure but it seems like you might have to enhance the `Session` model to store information like client agents to support this use case. The Auth part should stay the same.
